src/components/HomepageFeatures.js

```diff
@@ -4,48 +4,48 @@ import ReactMarkdown from "react-markdown";
+    content: "Add or remove nodes and let Pulsar load balance topic bundles automatically. Hot spotted topic bundles are automatically split and evenly distributed across the brokers.",
   },
   {
     title: "Serverless functions",
     Svg: require("../../static/img/proven-in-production.svg").default,
-    content:"Write serverless functions with developer-friendly APIs to natively process data immediately upon arrival. No need to run your own stream processing engine.",
+    content:"Write and deploy functions natively using Pulsar Functions. Process messages using Java, Go, or Python without deploying fully-fledged applications. Kubernetes runtime is bundled.",
   },
   {
-    title: "Horizontally scalable",
+    title: "Rapid Horizontal Scalability",
     Svg: require("../../static/img/horizontally-scalable.svg").default,
-    content: "Expand capacity seamlessly to hundreds of nodes.",
+    content: "Scales horizontally to handle the increased load. Its unique design and separate storage layer enable handling the sudden surge in traffic by scaling out in seconds.",
   },
   {
-    title: "Low latency with durability",
+    title: "Low-latency messaging and streaming",
     Svg: require("../../static/img/low-latency.svg").default,
-    content:"Low publish latency (< 5ms) at scale with strong durability guarantees.",
+    content:"Acknowledge messages individually (RabbitMQ style) or cumulative per partition (i.e., offset-like). Enables use cases such as distributed work queues or order-preserving data streams at very large scales (hundreds of nodes) and low latency (<5ms).",
   },
   {
-    title: "Geo-replication",
+    title: "Seamless Geo-Replication",
     Svg: require("../../static/img/geo-replication.svg").default,
-    content: "Configurable replication between data centers across multiple geographic regions.",
+    content: "Protect against complete zone outages using replication across different geographic regions. Flexible and configurable replication strategies across distant Pulsar Clusters. Uniquely supports automatic client failover to healthy clusters.",
   },
   {
     title: "Multi-tenancy",
     Svg: require("../../static/img/multi-tenancy.svg").default,
-    content:"Built from the ground up as a multi-tenant system. Supports isolation, authentication, authorization, and quotas.",
+    content:"Maintain one cluster for your entire organization using tenants. Access control across data and actions using tenant policies. Isolate specific brokers to a tenant when maximum noisy neighbor protection is needed.",
   },
   {
-    title: "Persistent storage",
+    title: "Official multi-language support",
     Svg: require("../../static/img/persistent-storage.svg").default,
-    content:"Persistent message storage based on Apache BookKeeper. IO-level isolation between write, and read operations.",
+    content:"Officially maintained Pulsar Clients for Java, Go, Python, C++, Node.js, and C#.",
```

Comment:
```suggestion
    content:"Officially maintain Pulsar Clients for Java, Go, Python, C++, Node.js, and C#.",
```
Use the present tense?